.github/workflows/publish-on-release.yml

```
@@ -0,0 +1,29 @@
+name: Publish Cookie Policy release
+
+# Only for testing. Should be on: release: types: [published]
+on:
+  - push
+
+jobs:
+  build-and-publish:
+    name: Publish Cookie Policy to NPM
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 18
+          registry-url: https://registry.npmjs.org/
+      - name: Build Cookie Policy
+        run: |
+          yarn install
+          yarn build
+          yarn test
+      - if: ${{ !github.event.release.prerelease }}
+        run: npm publish
```

not sure if that's needed, but this package is scoped to `@canonical`, so it may need `--access public` param.
```suggestion
        run: npm publish --access public
```

https://docs.npmjs.com/creating-and-publishing-scoped-public-packages#publishing-scoped-public-packages

---

I agree, great find!